backend/molgenis-emx2-fairdatapoint/src/main/java/org/molgenis/emx2/fairdatapoint/FAIRDataPointDistribution.java

```diff
@@ -127,7 +127,6 @@ public FAIRDataPointDistribution(Request request, Database database) throws Exce
 
     if (refersToTable) {
       if (formatParam.equals("csv")
-          || formatParam.equals("jsonld")
```

Comment:
Are we no longer supporting jsonld?

Reply:
we are, but that is all part of RDF content negotiation now